python/triton/runtime/jit.py

```diff
@@ -449,43 +449,6 @@ class JITFunction(KernelInterface[T]):
     # cache_hook will always be called before compilation and compiled_hook after.
     compiled_hook = None
 
-    @staticmethod
```

Comment:
Is it replaced by the `mangle_type` function?

Reply:
yup